src/flamenco/types/gen_stubs.py

```diff
@@ -546,7 +546,7 @@ def __init__(self, container, json):
         self.name = json["name"]
         self.element = json["element"]
         self.compact = ("modifier" in json and json["modifier"] == "compact")
-        self.max = (json["max"] if "max" in json else None)
+        self.min = json.get("min", None)
```

Comment:
Do you mind copying the explanation from the PR description as to why the deserialisation is provably bounded, so that future developers are cognisant of this?

Reply:
Maybe in `gen_stubs.py`? Or in fd_types.h, once. A comment in gen_stubs is probably fine though :)